hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -56,6 +77,18 @@ public OverwriteWithLatestAvroPayload preCombine(OverwriteWithLatestAvroPayload
     }
   }
 
+  public GenericRecord combineAllFields(List<Schema.Field> fields,GenericRecord priorRec,GenericRecord inferiorRoc) {
```

Comment:
we should def clean up variable names like `inferior`, in the interest of not having any naming with negative connotations. :)

Reply:
OK :>